ui.js

```diff
@@ -108,7 +108,7 @@
   DOM.on($el.togglePlay, 'click', () => {
     tp.config.play = !tp.config.play;
     $el.togglePlay.innerText = tp.config.play ? 'Stop' : 'Start';
-    tp.draw();
+    //tp.draw();
```

Comment:
I think that would be better if we remove this line completely?

Reply:
And BTW that would be great if you can add `:ambulance:` at the beginning of your commit message to stick with [gitmoji](https://gitmoji.carloscuesta.me)! :)